drivers/regex/mlx5/mlx5_regex.c

```diff
@@ -3,3 +3,15 @@
  */
 
 #include "mlx5_regex.h"
+
+static int mlx5_regex_is_supported(struct ibv_context *ibv_ctx)
```

Comment:
do we have the bool data type in  dpdk?
if so, please return as bool

Reply:
There are some cases of bool usage in DPDK but not many
I prefer int so we can use it as error code,